environs/config/config.go

```diff
@@ -176,6 +177,10 @@ const (
 	// FanConfig defines the configuration for FAN network running in the model.
 	FanConfig = "fan-config"
 
+	// CloudInitUserDataKey is the key to specify cloud-init yaml the user wants to add
+	// into the cloud-config juju produces for new machines.
```

Comment:
... into the cloud-config data produced by Juju when provisioning machines.

Reply:
updated